doc/source/changes/version_0_28.rst.inc

```diff
@@ -132,12 +132,11 @@ New features
 * made the editor able to detect and display plot objects stored in tuple, list or arrays.
   For example, arrays of plot objects are returned when using `subplots=True` option in calls of `plot` method:
 
-    >>> a = ndtest('sex=M,F;year=2000..2017')
-    >>> p = a.plot(subplots=True)
-    >>> type(p)
-    numpy.ndarray
-    >>> type(p[0])
-    matplotlib.axes._subplots.AxesSubplot
+    >>> a = ndtest('sex=M,F; nat=BE,FO; year=2000..2017')
+    >>> # display 4 plots vertically placed (one plot for each pair (sex, nationality))
+    >>> a.plot(subplots=True)
+    >>> # display 4 plots ordered in a 2 x 2 grid
+    >>> a.plot(subplots=True, layout=(2, 2))
```

Comment:
this is much better IMO, as it is both an interesting feature and related to what was changed